sale_order_delivered_weight/models/sale_order.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2021 Manuel Calero Solís (https://xtendoo.es)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    total_delivered_weight = fields.Float(
+        compute='_compute_total_delivered_weight',
+        string='Total Delivered Weight',
+        store=True
+    )
+
+    @api.depends('order_line.qty_delivered')
```

Comment:
Do miss "order_line.product_id.weight" ?

Reply:
@rousseldenis this module has been replaced by :

https://github.com/OCA/sale-workflow/pull/1557